src/core/Cline.ts

```diff
@@ -2568,13 +2568,20 @@ export class Cline {
 									?.mcpHub?.callTool(server_name, tool_name, parsedArguments)
 
 								// TODO: add progress indicator and ability to parse images and non-text responses
+								const supportsImages = this.api.getModel().info.supportsImages ?? false
```

Comment:
Whether the model supports images or not shouldn't matter here, since MCP servers can always output base64 either way and the model would see the string representation.